docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -83,6 +83,28 @@ API ML provides a Zowe CLI plugin to help administrators generate a JCL for crea
 
   See the [Identity Federation cli plugin](../../user-guide/cli-idfplugin.md) documentation for details about how to use the plugin tool to set up the mapping in the ESM of your z/OS system.
 
```

Comment:
Please rephrase to have first the commands description and then mention the tool as a way to make it easier